script/main.js

```diff
@@ -0,0 +1,44 @@
+(function(){
+    function getRandomInt (min, max) {
+        min = Math.floor(min);
+        max = Math.ceil(max);
+        return Math.floor(Math.random() * (max - min + 1)) + min;
+    }
+
+    let firstDice={
+        number:getRandomInt(1, 6),
+        roll: () => {
+            this.number = getRandomInt(1, 6);
+            return this.number;
+        }
+    };
+
+    let scundDice={
+        number:getRandomInt(1, 6),
+        roll: () => {
+            this.number = getRandomInt(1, 6);
+            return this.number;
+        }
+    };
+
+    function imagesToPrint (img1, img2) {
+        let firstImag = document.querySelector('#first-dice');
+        let scundImg = document.querySelector('#scund-dice');
+        firstImag.style['background-image'] = `url(images/inverted-dice-${img1}.svg)`;
+        scundImg.style['background-image'] = `url(images/inverted-dice-${img2}.svg)`;
+    }
+
+    let button = document.querySelector('#button-roll');
+    button.addEventListener('click', event => {
+        event.preventDefault();
+        let firstroll = firstDice.roll();
```

Comment:
You don't actually have to store the return of the dice into a variable. You can also just roll the dice and then pass `number` property from the dice object into the `imagesToPrint()` function below, since the role method **actually** changes the `number` property on the dice object. Like this:

```javascript
firstDice.roll();
scundDice.roll();

imagesToPrint(firstDice.number, scundDice.number);
```